README.md

```diff
@@ -162,16 +143,31 @@ webExt.cmd.run({
   });
 ```
 
-To get a list of connected adb devices, you may use the following function which takes an optional argument `adbBin` which is  a path to custom adb binary:
+#### Running the extension on Firefox for android:
 ```js
-webExt.util.adb.listADBDevices(); // returns a Promise<Array<string>>
-```
+// Path to adb binary (optional parameter, auto-detected if missing)
+const adbBin = "/path/to/adb";
+// Get an array of device ids (Array<string>)
+const deviceIds = await webExt.util.adb.listADBDevices(adbBin);
+const adbDevice = ...
+// Get an array of Firefox APKs (Array<string>)
+const firefoxAPKs = await webExt.util.adb.listADBFirefoxAPKs(
+  deviceId, adbBin
+);
+const firefoxApk = ...
 
-To get a list of installed firefox APK packages for an ADB client, you may use the following function which takes in deviceId(required) and adbBin(optional) as arguments:
-```js
-webExt.util.adb.listADBFirefoxAPKs(deviceId); // returns a Promise<Array<string>>
+webExt.cmd.run({
+  target: 'firefox-android',
+  firefoxApk,
+  adbDevice,
+  sourceDir: ...
+}).then((extensionRunner) => {...});
 ```
 
+
+
+
+
```

Comment:
One empty line between the end of the code snippet and the start of the paragraph that follows is enough, let's remove the additional empty lines.